### ISSUE TITLE
can't install from the git repo with OpenJDK

### ISSUE THREAD

**Issue by jeanbaptisteb** [myint/language-check#54]
Config: 
- Ubuntu Server 18.04 "Bionic Beaver" LTS (64bits) 
- Python 3.6.5, 
- openjdk 10.0.1 2018-04-17
- OpenJDK Runtime Environment (build 10.0.1+10-Ubuntu-3ubuntu1)
- OpenJDK 64-Bit Server VM (build 10.0.1+10-Ubuntu-3ubuntu1, mixed mode)

I tried to install language-check from pip, and the installation works. But then, when I try using it in some Python code, it throws some error. 

So I tried to install language-check directly from the git repo, as suggested in another ticket, with:
`pip3 install git+https://github.com/myint/language-check.git`
But then I get:

>  Could not parse Java version from """openjdk version "10.0.1" 2018-04-17
>  OpenJDK Runtime Environment (build 10.0.1+10-Ubuntu-3ubuntu1)
>  OpenJDK 64-Bit Server VM (build 10.0.1+10-Ubuntu-3ubuntu1, mixed mode

I also tried installing it on Windows, with the following configuration, and in this case it works like a charm:

- Windows 10, 64 bits, 
- Python 3.6.3
- java version "1.8.0_171"
- Java(TM) SE Runtime Environment (build 1.8.0_171-b11)
- Java HotSpot(TM) 64-Bit Server VM (build 25.171-b11, mixed mode) 

So I guess that the problem comes from OpenJDK on Ubuntu. 

Any advice?

**Comment by Bachstelze** [comment]
Same issue here with the 18.04 ubuntu desktop image.
Got this error message:

  Running setup.py bdist_wheel for language-check ... error
  Complete output from command /home/bag_of_words/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-w08w5t2o/language-check/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/pip-wheel-wap2l03f --python-tag cp36:
  Could not parse Java version from """openjdk version "10.0.1" 2018-04-17
  OpenJDK Runtime Environment (build 10.0.1+10-Ubuntu-3ubuntu1)
  OpenJDK 64-Bit Server VM (build 10.0.1+10-Ubuntu-3ubuntu1, mixed mode)

**Comment by jeanbaptisteb** [comment]
It seems that the culprit is here: https://github.com/myint/language-check/blob/80b0dc02c77a5adf616157342883fd84d97e22ba/download_lt.py#L85 (versions of OpenJDK > 8 won't work). But if the library hasn't been tested yet against versions of OpenJDK > 8, updating this line might not solve the issue!

**Comment by Bachstelze** [comment]
Yes, should be a duplication because they also can not parse the java version with this regex, but it is not failing with only one certain java version.
You can try my Pull-request or delete on your own 3 characters :D

**Comment by Bachstelze** [comment]
This should be fixed with https://github.com/myint/language-check/pull/57
Can we make a new release for pip with this bug fix?

**Comment by jxmorris12** [comment]
Hi! Since this project has been abandoned, I started a new fork over at https://github.com/jxmorris12/language_tool_python. 

My version supports new versions of Java and LanguageTool. `language-check` is stuck on Java 8 and LanguageTool 3.2; latest versions are Java 14 and LanguageTool 4.9!

I'm happy to help you with your issue if you raise it over at my repository! Thanks!
